src/offsetPath.js

```diff
@@ -13,9 +45,10 @@
 
     var ray = /^ray\((.*)\)$/.exec(input);
```

Comment:
Code clean up idea:

    function offsetPathParse(input) {
      for (parse of [parseNone, parseRay, parsePath, parseCircle...]) {
        var result = parse(input);
        if (result) {
          return result;
        }
      }
    }